BnbnavNetClient/Services/EditControllers/EditController.cs

```diff
@@ -1,20 +1,19 @@
-using System.Collections.Generic;
 using Avalonia.Input;
 using Avalonia.Media;
 using BnbnavNetClient.Models;
 using BnbnavNetClient.Views;
 
 namespace BnbnavNetClient.Services.EditControllers;
 
-public enum PointerPressedFlags
+public enum PointerPressed
```

Comment:
This was a bitflag - I think changing the name might affect our understanding of the code later. What do you think?

Reply:
[CA1711](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1711) says the suffix Flags should not be used for enums, even if they were bitflags, following [design guidelines](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces).